### PR TITLE
Use maven central

### PR DESCRIPTION
**Pull request by tjcarroll11**
Updates all CST examples to point to maven central for dependencies

### CODE REVIEW

File: Connected System Template (CST) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/.idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="11" project-jdk-type="JavaSDK">
```
Comment (tjcarroll11) [inline Connected System Template (CST) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/.idea/misc.xml]: Fix this

File: Connected System Template (CST) Examples/Hello World/.idea/misc.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="11" project-jdk-type="JavaSDK">
```
Comment (tjcarroll11) [inline Connected System Template (CST) Examples/Hello World/.idea/misc.xml]: fid this

File: Client API enabled CSE Examples/textDetection/cst/.idea/misc.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="11" project-jdk-type="JavaSDK">
```
Comment (tjcarroll11) [inline Client API enabled CSE Examples/textDetection/cst/.idea/misc.xml]: fix

File: Client API enabled CSE Examples/textDetection/cst/.idea/gradle.xml [hunk 4]
```diff
@@ -3,9 +3,9 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="disableWrapperSourceDistributionNotification" value="true" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="gradleJvm" value="#JAVA_HOME" />
```
Comment (benjamin-kaiser) [inline Client API enabled CSE Examples/textDetection/cst/.idea/gradle.xml]: we determined we needed this file?

**Pull request merged**